app/database_etl/etl_main.py

```diff
@@ -203,21 +220,49 @@ def create_bridge_tables(original_data, multi_select_tables):
 def load_postgres_tables(tables_dict, engine):
     # Load dataframes into postgres tables
     for table_name, table_value in tables_dict.items():
-        table_value.to_sql(table_name,
-                           schema='public',
-                           con=engine,
-                           if_exists='append',
-                           index=False)
+        if table_name == 'research_source':
+            table_value.loc[table_value['estimate_name'] == '0724_DeKalbFulton_CDCCOVID-19ResponseTeam_Female_PopAdj',
+                            'jbi_2'] = 'somesuperlongstringthatwillcauseitalltofailhahahahhahahyessssss'
+        try:
+            table_value.to_sql(table_name,
+                               schema='public',
+                               con=engine,
+                               if_exists='append',
+                               index=False)
+            drop_old_entries(table_name)
+        except (SQLAlchemyError, ValueError) as e:
+            # Send error email
+            logging.error(e)
+            # TODO: send slack message with error
+
+            # Delete  records in table that failed which contain current datetime (remove records from current ETL run)
```

Comment:
hmm should we just fully revert to the pre etl state if we see a failure, and not just revert the current table?

reason I'm suggesting this is because if table A is reverted but not table B, then the bridge table between table A and B might be fucked up

Reply:
mmm that is a very good point